sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/CounterSourceConnector.java

```diff
@@ -1,3 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I guess we just remove this file, right? since it's not part of what we need for implementation.

Reply:
or perhaps - instead of simply removing the file, let's move the class to a `*Test.java` file where we can use it to test the kafka consumer Splittable DoFn.